charmcraft/templates/init/src/charm.py.j2

```diff
@@ -2,7 +2,51 @@
 # Copyright {{ year }} {{ author }}
 # See LICENSE file for licensing details.
 
-"""Charm the service."""
+"""Charm the service.
+
+To gently guide you through the recommended way to write and deploy your new charm,
+here are some essential links and instructions to get you up-to-speed.
```

Comment:
These instructions are k8s-specific, and while the default charm is created for k8s, the Operator Framework can equally be used to write machine charms, so we should call out specifically that these instructions are for k8s charming.

Also, might be worth including a links for where to get help, although this could be a duplication with https://github.com/canonical/operator#talk-to-us, so maybe just refer to that?

Reply:
Good point! I will add that and the talk-to-us link.